header-gossip-network-test-vectors.md

```diff
@@ -0,0 +1,38 @@
+# Header Gossip Network Test Vectors
+
+This document contains test vectors for the Header Gossip Network to help implementations conform to the specification.
+
+## Header Accumulator Encodings
+
+These test vectors specify the proper encoding of the header accumulator in certain well defined scenarios
+
+### Epoch 0 Accumulator
+
+#### Accumulator after genesis block
+
+This test vector represents the input parameters and the expected output for the current epoch accumulator after adding
+the genesis block from Ethereum Mainnet.
+
+##### Input Parameters
```

Comment:
Considering that the `update_accumulator` call in the specs takes the `BlockHeader` as parameter, should we not provide the full block header as input parameters?

Reply:
Either way. My actual code does take the full header as input so was just putting the header record as that's the actual input to the accumulator update itself. Will update this as well.

Reply:
Have switched out the inputs for the block rlps

Reply:
I think the genesis rlp header is incorrect. The one I am using is `0xf90214a00000000000000000000000000000000000000000000000000000000000000000a01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347940000000000000000000000000000000000000000a0d7f8974fb5ac78d9ac099b9ad5018bedc2ce0a72dad1827a1709da30580f0544a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421a056e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421b9010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000850400000000808213888080a011bbe8db4e347b4e8c937c1c8370e4b5ed33adb3db69cbdb7a38e1e50b1b82faa00000000000000000000000000000000000000000000000000000000000000000880000000000000042`. This gives me the proper hash of `0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3`

I'm also getting different hash roots with your provided genesis header though

Reply:
With my latest updates, hash tree roots should match what's in your Fluffy tests